### DISCUSSION TITLE
Add support for NDS database for Navigation DB

### DISCUSSION THREAD

**Discussion by jitendrarupeja** [sqlitebrowser/sqlitebrowser#3152] [General]
Add support for NDS database for Navigation DB

**Comment by chrisjlocke** [comment]
An NDS database is an sqlite database, so DB4S should be able to open it.
Do you mean support as in import from or export to?  There are a number of NDS specific tools (map viewers, etc) so don't see how 'supporting' it in DB4S would be beneficial?  Their tools, more built for the job would be suitable, surely?

**Comment by chrisjlocke** [comment]
Switching to a discussion, as this isn't an issue, per se.
Can be switched back if need be.
